tests/src/main/kotlin/se/ansman/kotshi/Wildcards.kt

```diff
@@ -1,7 +1,7 @@
 package se.ansman.kotshi
 
 @JsonSerializable
-data class Wildcards(val list: List<CharSequence>) {
+data class Wildcards(var list: List<CharSequence>) {
```

Comment:
Won't this be `List<? extends Charsequence>` and fail in the next version of Moshi?
Don't we need to normalize to the upper bound, or did I miss something?

Reply:
We already do, the generated code looks like this:
```kotlin
  private final JsonAdapter<List<? extends CharSequence>> adapter0;

  public KotshiWildcardsJsonAdapter(Moshi moshi) {
    super("KotshiJsonAdapter(Wildcards)");
    adapter0 = moshi.adapter(Types.newParameterizedType(List.class, CharSequence.class));
  }
```

Reply:
oh, nice. i missed that in AdapterKey.
LGTM.